### PR TITLE
CMR-6257 - NGDA Tags

### PR DESCRIPTION
**Pull request by jceaser**
Changed the rules on how NGDA tags are added to the opendata output. This change adds instrument data to the metadata used by the open data results handler. These values are then used to drive the inclusion of the NGDA tags. Instruments are not included in the opendata output.

### CODE REVIEW

File: search-app/src/cmr/search/results_handlers/opendata_results_handler.clj [hunk 1]
```diff
@@ -242,11 +245,24 @@
     {:name (or archive-center umm-spec-util/not-provided)
      :subOrganizationOf hierarchy}))
 
+(defn check_for_modis_aster
```
Comment (jaybarra) [inline search-app/src/cmr/search/results_handlers/opendata_results_handler.clj:248]: Is this going to be such a unique case that we won't need to generalize this to a multi-method?

Comment (jaybarra) [reply search-app/src/cmr/search/results_handlers/opendata_results_handler.clj:248]: Also please rename to use dashes to be inline with code conventions. Suggestion **modis-aster?**

**Review by mitchstartzel — APPROVED** [review]

File: search-app/src/cmr/search/results_handlers/opendata_results_handler.clj (hunk 1 above)
Comment (jceaser) [reply search-app/src/cmr/search/results_handlers/opendata_results_handler.clj:248]: I think the long term solution is to move this logic to cmr tags and not drive it through business logic. At least that is what Dana has suggested. Until then I think it is unique.

File: search-app/src/cmr/search/results_handlers/opendata_results_handler.clj [hunk 2]
```diff
@@ -242,11 +245,24 @@
...
+  "Logic to test for collections that use the MODIS or ASTER instruments as
+  these are the only ones that should be included in the data.gov output as
+  defined by The Data Curation for Discovery (DCD) team at Marshall Space Flight
+  Center (MSFC)"
+  [x]
+  (let [short-name (:short-name x)]
```
Comment (jaybarra) [inline search-app/src/cmr/search/results_handlers/opendata_results_handler.clj:254]: Prefer destructured syntax and more verbose input value than x
'let [{:keys [short-name]} collection]`

File: search-app/src/cmr/search/results_handlers/opendata_results_handler.clj [hunk 3]
```diff
@@ -242,11 +245,24 @@
     {:name (or archive-center umm-spec-util/not-provided)
      :subOrganizationOf hierarchy}))
 
+(defn check_for_modis_aster
+  "Logic to test for collections that use the MODIS or ASTER instruments as
+  these are the only ones that should be included in the data.gov output as
+  defined by The Data Curation for Discovery (DCD) team at Marshall Space Flight
+  Center (MSFC)"
+  [x]
```
Comment (mschmele) [inline search-app/src/cmr/search/results_handlers/opendata_results_handler.clj:253]: `x` -> `collection` or something more descriptive

**Review by jaybarra — CHANGES_REQUESTED** [review]

File: search-app/src/cmr/search/results_handlers/opendata_results_handler.clj (hunk 1 above)
Comment (jceaser) [reply search-app/src/cmr/search/results_handlers/opendata_results_handler.clj]: @jaybarra - do you have issue with this still?

**Review by jaybarra — APPROVED** [review]

**Review by mschmele — APPROVED** [review]

**Pull request merged**